R/filter_df_for_metamean.R

```diff
@@ -0,0 +1,71 @@
+#' Prepare parameter dataframe for meta analysis of means
+#'
+#' @param df a parameter dataframe. This must have columns for each of the
+#' following: parameter_value, parameter_unit, population_sample_size,
+#' parameter_value_type, parameter_uncertainty_singe_type,
+#' parameter_uncertainty_type, parameter_uncertainty_lower_value,
+#' parameter_uncertainty_upper_value
+#'
+#' @return a parameter dataframe with relevant rows selected
+#'
+#' @details The function checks that the format of df is adequate for conducting
```

Comment:
I think we are going to have to add details on what we mean by relevant and what is being filtered out so that the documentation is self-contained.